packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -131,7 +132,7 @@ export const getOptionStyles = memoizeFunction(
       },
       rootChecked: [
         {
-          backgroundColor: 'transparent',
+          backgroundColor: option.backgroundSelectedColor,

```

Comment:
LGTM - menuItemBackgroundChecked is in line with the other menuitem states root is being set to here